bin/setupReplication.js

```diff
@@ -62,23 +66,36 @@ function _setupS3Client(transport, endpoint, profile) {
         endpoint: `${transport}://${endpoint}`,
         sslEnabled: transport === 'https',
         credentials,
+        region: 'us-east-1',
```

Comment:
Changed to `us-east-1`. Saw `us-east-2` was not defined in `/conf/locationConfig.json` in S3 server